PageSpeed/Test/NewTestViewController.swift

```diff
@@ -25,6 +25,7 @@ class NewTestViewController: UIViewController {
     var mobilePageSpeedResult: PageSpeedResponse?
     var desktopPageSpeedResult: PageSpeedResponse?
     var gtMetrixResponse: GTMetrixResponseItem?
+    var progressView: ProgressView?
```

Comment:
@ilyageekhub why **ProgressView** is optional? Also, why you declare global variable that's only used in **processRequestsToServices**? If so, you should remove this global variable and use only local one.

Reply:
fix